src/backend/oracle-data-api/src/main/java/ca/bc/gov/open/jag/tco/oracledataapi/mapper/DisputeMapper.java

```diff
@@ -14,22 +14,24 @@
 import ca.bc.gov.open.jag.tco.oracledataapi.api.model.ViolationTicket;
 import ca.bc.gov.open.jag.tco.oracledataapi.model.Dispute;
 import ca.bc.gov.open.jag.tco.oracledataapi.model.DisputeCount;
+import ca.bc.gov.open.jag.tco.oracledataapi.model.DisputeStatus;
 import ca.bc.gov.open.jag.tco.oracledataapi.model.Plea;
 import ca.bc.gov.open.jag.tco.oracledataapi.model.ViolationTicketCount;
 import ca.bc.gov.open.jag.tco.oracledataapi.model.YesNo;
+import ca.bc.gov.open.jag.tco.oracledataapi.repository.impl.ords.DisputeRepositoryImpl;
 
 @Mapper
 public interface DisputeMapper {
-	
+
 	DisputeMapper INSTANCE = Mappers.getMapper(DisputeMapper.class);
-	
+
 	// Map dispute data from ORDS to Oracle Data API dispute model
-	@Mapping(source = "dispute.entDtm", target = "createdTs")
+	@Mapping(source = "dispute.entDtm", target = "createdTs", dateFormat = DisputeRepositoryImpl.DATE_TIME_FORMAT)
```

Comment:
This was needed to parse the dateTime string coming from ORDS. This appears to be "yyyy-MM-dd HH:mm:ss"

Reply:
Yeah, I was surprised by this as well. I thought you had the dates all working, I remember reviewing a PR with it. Was the code in main or in another branch?  Either way, I kept getting mapper errors with the date fields without this. You're welcome to fix this after I merge this into main.